aea/contracts/base.py

```diff
@@ -43,15 +43,15 @@ class Contract(Component, ABC):
     """Abstract definition of a contract."""
 
     def __init__(
-        self, config: ContractConfig, contract_interface: Dict[str, Any],
+        self, contract_config: ContractConfig, contract_interface: Dict[str, Any],
```

Comment:
made the parameter name consistent across contract classes. This is useful to pass keyword arguments when registering the item (see line 1408 and 1409 in `aea/aea_builder.py`). However, this could be break by the user that renames the arguments in the constructor of the concrete contract.